src/main/java/net/snowflake/client/jdbc/SnowflakeConnectString.java

```diff
@@ -154,7 +154,7 @@ public static SnowflakeConnectString parse(String url, Properties info) {
 
       return new SnowflakeConnectString(scheme, host, port, parameters, account);
     } catch (Exception ex) {
-      logger.info("Exception thrown while parsing Snowflake connect string", ex);
+      logger.warn("Exception thrown while parsing Snowflake connect string", ex);
```

Comment:
This isn't this PR's fault so shouldn't be a blocker, but I'm throwing it out there as just an FYI @sfc-gh-igarish as part of that discussion we had around exception handling and logging. I'm a bit uneasy about the fact that we're catching all exceptions and then simply printing out a WARN message (if logging is enabled). I don't immediately see anything that would cause a significant exception, but can you also take another glance?

Reply:
My opinion would be that, for this method, we are fine. This function is used to judge whether a connection string is valid or not. If we hit any exception during the parsing, it makes sense that we treat it as invalid. And the reason for invalid string can be found with logging enabled.